recipes/gneiss/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: gneiss
+  version: 0.1.0
+
+source:
+  git_url: https://github.com/biocore/gneiss.git
+
+
+requirements:
+  build:
+    - python
+    - scikit-bio ==0.4.2
+    - pandas
+    - scipy
+    - numpy
+    - matplotlib
+    - pyqt
```

Comment:
Can you limit this to pyqt <= 4.11.3? It has to do with licensing, pyqt has exceptions for BSD in 4.11.3 but removed them in 4.11.4